doc/nrf/includes/sample_board_rows.txt

```diff
@@ -248,6 +248,10 @@
 
 | nRF54L20 PDK |   | :zephyr:board:`nrf54l20pdk` | ``nrf54l20pdk/nrf54l20/cpuapp`` |
 
+.. nrf54lv10apdk_nrf54lv10a_cpuapp
+
```

Comment:
Just noticed but the build is still failing for the samples, target needs to be fixed

Reply:
Seems they have decided on renaming it again, to nrf54lv10dk. So going to wait a bit with this PR until that lands